event-processing/src/test/resources/eventHandler.bpmn

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:zeebe="http://camunda.org/schema/zeebe/1.0" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="Definitions_083ntxc" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="5.29.0" modeler:executionPlatform="Camunda Cloud" modeler:executionPlatformVersion="8.6.0">
+  <bpmn:process id="eventHandler" name="Event Handler" isExecutable="true">
+    <bpmn:startEvent id="StartEvent_1" name="Event received">
+      <bpmn:extensionElements>
+        <zeebe:executionListeners>
+          <zeebe:executionListener eventType="end" type="correlated" />
```

Comment:
Event listeners are information hiding. I prefer a service task to show and name the activity.

Reply:
For an example, we should add some business logic here.

Reply:
The information hiding is done on purpose. The callback to the event sender is not related to any business functionality.

I can add some business logic here of course :)